test/pageobjects/DocumentUploadConfirmation.js

```diff
@@ -79,9 +81,15 @@ class DocumentUploadConfirmation extends Base {
     verifyElementCopy(this.errorInstruction, documentUploadConfirmationErrorStrings.glare_detected.instruction)
   }
 
+  // TODO: this function is never called!
```

Comment:
Let's remove it.

Reply:
No, I started using it as part of this PR. I need to remove the comment and make sure I refactor the other tests to use this function :)